utils/src/main/scala/za/co/absa/enceladus/utils/general/JsonUtils.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.utils.general
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+object JsonUtils {
```

Comment:
If you don't have test, I would at least append 2 sentences about what each is and why is it created.